### ISSUE TITLE
Ignore interpolated values for misspelled properties

### ISSUE THREAD

**Issue by stowball** [sasstools/sass-lint#720]
When linting for misspellings, interpolated values within property names warn of a spelling mistake. It would be good to exclude these somehow.

For instance

``` scss
$u-layout-divider-types: (
    bottom,
    left,
    right,
    top
);

@each $type in $u-layout-divider-types {
    .u-layout-divider-#{$type} {
        border-#{$type}: 1px solid #000 !important;
    }
}
```

results in "Property `border-` appears to be spelled incorrectly"

Using v1.7.0 of sass-lint through gulp-sass

**Comment by DanPurdy** [comment]
This is 'fixed' in 1.8.0 but only because I've spotted a new bug! 😭
